types/prompts/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for prompts 2.04
+// Type definitions for prompts 2.0.4
```

Comment:
`prompts` is currently available as 2.0.4: https://www.npmjs.com/package/prompts

Reply:
Not sure if changing the version here like this is a "breaking change" in `DefinitelyTyped`, though.

Reply:
(Updated to `2.0`)